de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolImage.java

```diff
@@ -19,4 +19,10 @@
 
     @PropertyName("url")
     String getUrl();
+
+    @PropertyName("auth")
+    String getAuth();
+
+    @PropertyName("auth")
+    void setAuth(String auth);
```

Comment:
I don't think this field will ever be sent to the UI, except maybe to the admin UI in the future.
It doesn't look like this field is accessed in this PR either, so I don't think we need to add it.